packages/button/tests/ButtonBase.test.tsx

```diff
@@ -17,7 +17,7 @@ describe("ButtonBase", () => {
           appearance={ButtonAppearances[appearance]}
           isFullWidth={true}
           type="submit"
-          ariaHaspopup={true}
+          aria-haspopup={true}
```

Comment:
I'm new, so i ask dumb questions. What is changing (or not changing) due to this change?

My best bet would be that before `ButtonBase` was in charge of rendering this prop and now we directly let `button` do this? If so, where can i see that this is working? With a brief look at the snapshots i don't find that property. help!

Reply:
That's odd - I'm not sure why this wasn't showing up in previous snapshots.

You're currect though, it changes because now we let `button` do this directly, so instead of using `ariaHaspopup`, we have to use `aria-haspopup`